apps/webapp/app/hooks/useKapaWidget.tsx

```
@@ -0,0 +1,143 @@
+        data-website-id={websiteId}
+        data-project-name={"Trigger.dev"}
+        data-project-color={"#6366F1"}
+        data-project-logo={"https://content.trigger.dev/trigger-logo-circle.png"}
+        data-render-on-load={"false"}
+        data-button-hide={"true"}
+        data-modal-disclaimer-bg-color={"#D7D9DD"}
+        data-modal-header-bg-color={"#2C3034"}
+        data-modal-title-color={"#D7D9DD"}
+      ></script>
+      <script
+        dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)